class/Game.js

```diff
@@ -22,7 +22,8 @@ export class Game {
         this.lastTime = performance.now();
         this.timer = performance.now();
         this.ms = 1000 / GAME_UPS;
-        this.delta = 0;
```

Comment:
I swapped the names of `this.delta` and `this.progress`, because I think delta should be the time between frames, and progress should be the "progress" toward the next update